docs/fiddles/accelerator/main.js

```diff
@@ -0,0 +1,25 @@
+// Adding a global shortcut to electron
+//
+// For more info, see:
+// https://electronjs.org/docs/api/screen
+
+const { app, globalShortcut, BrowserWindow } = require('electron')
+
+let mainWindow = null;
+
+app.on('ready', () => {
+  // Register a 'CommandOrControl+Y' shortcut listener.
+  globalShortcut.register('CommandOrControl+Y', () => {
+    // Do stuff when Y and either Command/Control is pressed.
+    console.log('shortcut...');
+  });
+  
+  mainWindow = new BrowserWindow({
+    width: 800,
+    height: 600
+  });
+
+  // and load the index.html of the app.
+  mainWindow.loadFile('index.html');
```

Comment:
hmm there is no `index.html` in this fiddle - we can remove this line or add a simple html